gui/qt/fee_slider.py

```diff
@@ -48,3 +51,15 @@ def update(self):
             self.setValue(pos)
             tooltip = self.get_tooltip(pos, fee_rate)
             self.setToolTip(tooltip)
+
+    # configuraing this as is done is here still required, can't just set range 0,0 to deactivate.
+    # chose to make this a seperate function from update for easier code maintainence
+    def deactivate(self):
+        with self.lock:
+            self.fee_step = self.config.max_fee_rate() / 10
+            fee_rate = self.config.fee_per_kb()
+            pos = min(fee_rate / self.fee_step, 10)
+            self.setRange(0, 0)
+            self.setValue(pos)
+            tooltip = self.get_tooltip(pos, fee_rate)
+            self.setToolTip(tooltip)
```

Comment:
You should definitely call .hide() here.

On macOS setting the range to 0,0 causes the fee slider to disappear, which is fine and probably a good thing!

On Linux the fee slider does NOT disappear but is instead deactivated.  Calling 0,0 on a slider in Qt is undefined (it's platform-specific).

I suggest you either:

1. hide() it (and unhide it in update()) <-- this way is hidden on all platforms
2. or call setEnabled(False) and setRange(0,1).  <-- this way is deactivated on all platforms.

What you're doing here is going to produce inconsistent behavior.

on macOS: the fee slider is hidden
on Linux: the fee slider is not hidden but is grayed out.